packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/ScatterGraph/Reflow/SimplePanning.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Button from 'terra-button/lib/Button';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import '@cerner/terra-graphs/lib/components/Graph.module.scss';
+import getConfigLineTimeseriesPanning from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Line/lineTimeseriesPanning';
+import exampleData from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Line/timeseriesData';
+/*
+Please refer to the documentation below to see the graphConfig and data objects
+*/
+
+const graphConfig = utils.deepClone(getConfigLineTimeseriesPanning('#scatterPanningExample'));
+const dataset = utils.deepClone(exampleData);
+
+const initialState = {
+  initial: 0,
+  factor: 3,
+};
+let graph;
+
+const SimplePanningExample = () => {
+  React.useEffect(() => {
+    graph = Carbon.api.graph(graphConfig);
+    dataset.forEach((data) => {
+      graph.loadContent(Carbon.api.scatter(data));
+    });
+  }, []);
+
+  const reducer = (panState, action) => {
+    let hour;
+
+    switch (action.type) {
+      case 'panLeft':
+        hour = panState.initial - panState.factor;
+        break;
+      case 'panRight':
+        hour = panState.initial + panState.factor;
+        break;
+    }
+
+    graph.config.axis.x.lowerLimit = new Date(2016, 0, 1, hour).toISOString();
+    graph.config.axis.x.upperLimit = new Date(2016, 0, 2, hour).toISOString();
```

Comment:
Can remove 42, 43 now

Reply:
Yeah realized lately and removed it in the next [commit](https://github.com/cerner/terra-graphs/pull/194/commits/91e4839e0402cc8fd67ea792f3efc6c6bfef4860#diff-51aa3df6b894082f674c1535b2e1fe854986b25c285136985e89111ccb86d699L42).